VaticanBAV/borget/BAVborget9.xml

```diff
@@ -61,79 +61,29 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
 
                     <msItem xml:id="ms_i4">
                       <locus from="53v" to="64v" facs="0108"/>
-                      <title ref="LIT4315Miracles">Ten Miracles of St ʾEwosṭātewos in Verse.</title>
+                      <title ref="LIT6201Hymn"/>
                       <textLang mainLang="gez"/>
 
                       <msItem xml:id="ms_i4.1">
-                        <locus from="53v" to="54v"/>
-                        <title>Prologue</title>
+                        <locus from="53v" to="54v" facs="0108"/>
+                        <title ref="LIT6200Hymn"/>
                         <incipit xml:lang="gez">መንክራቲከ፡ ሰማዕነ፡ እንዘ፡ ትገብር፡ <supplied reason="omitted">በ</supplied>አርማንያ፨ መካነ፡ ኤዎስጣቴዎስ፡ ተአምር፡ በኢትዮጵያ፡</incipit>
                       </msItem>
 
                       <msItem xml:id="ms_i4.2">
-                        <locus from="54v" to="55r"/>
-                        <title>First Miracle</title><!-- Cf. LIT3053RepCh333 -->
-                        <incipit xml:lang="gez">እምብዙኅ፡ ተአምሪከ፡ ዕዜኑ፡ አሐተ፡ ኤዎስጣቴዎስ፡ ዘትገብር፨ ዓበይተ፡ ኃይላተ፡ መኰንን፡ ዝኁር፨ ሶበ፡ ፈቀደ፡ ነሢተ፡ መቃብሪከ፡ ክብርተ፨</incipit>
+                        <locus target="#54v" facs="0110"/>
+                        <title ref="LIT3043RepCh323"/>
                       </msItem>
 
                       <msItem xml:id="ms_i4.3">
-                        <locus target="#55r"/>
-                        <title>Second Miracle</title>
-                        <incipit xml:lang="gez">ካልዕ፡ ተአምሪከ፡ ኤዎስጣቴዎስ፡ ድንግል፡ አመ፡ ውስተ፡ ባሕር፡ ተሰጥመ፨ እንዘ፡ ይጐይይ፡ እምቀትል፤</incipit>
+                        <locus from="54v" to="58v" facs="0110"/>
+                        <title ref="LIT3053RepCh333"/>
+                        <incipit xml:lang="gez">እምብዙኅ፡ ተአምሪከ፡ ዕዜኑ፡ አሐተ፡ ኤዎስጣቴዎስ፡ ዘትገብር፨ ዓበይተ፡ ኃይላተ፡ መኰንን፡ ዝኁር፨ ሶበ፡ ፈቀደ፡ ነሢተ፡ መቃብሪከ፡ ክብርተ፨</incipit>
                       </msItem>
 
                       <msItem xml:id="ms_i4.4">
-                        <locus target="#55r"/>
-                        <title>Third Miracle</title>
-                        <incipit xml:lang="gez">ሣልስ፡ ተአምሪከ፡ ዝውእቱ፤ ከመ፡ አድኃንኮሙ፡ አባ፡ ለኃብተ፡ እግዚእ፡ ወለወለቱ፨ እመቅሠፍተ፡ ሥራይ፡ ኅሱም፡</incipit>
```

Comment:
the incipits of the stanzas, now suppressed, should go to LIT3053RepCh333 I assume?

Reply:
Yep! See https://github.com/BetaMasaheft/Works/blob/8db37e0225f3f06cd7e92ff743a91027c14d6009/3001-4000/LIT3053RepCh333.xml#L64